### ISSUE TITLE
Disabling the auto-clicking without disabling the blocking of ads for certain sites

### ISSUE THREAD

**Issue by TGSaK** [dhowe/AdNauseam#1130]
It seems to me that it isn't possible to turn off the the clicking part of the addon (for a single website) without losing the adblocking part at the same time, since pausing AdNauseam turns both of them off.

I think it would be a great addition if you could individually whitelist sites for both clicking and blocking.

**Comment by ajayyy** [comment]
this is needed

**Comment by GiovanH** [comment]
I specifically came here to make this request.

**Comment by dhowe** [comment]
Right. This is currently possible for the addon as a whole, but not for individual sites. Any thoughts on what an interface for this would look like, given our current menu?
